packages/dbml-parse/src/lib/parser/parser.ts

```diff
@@ -826,9 +826,7 @@ export default class Parser {
 
     throw new PartialParsingError(
       this.peek(),
-      this.nodeFactory.create(PrimaryExpressionNode, {
-        expression: this.nodeFactory.create(VariableNode, {}),
-      }),
```

Comment:
Please explain this change and why the previous code didn't work

Reply:
I have added here in the description anh:
![image](https://github.com/holistics/dbml/assets/139191192/16dd9a1b-bc80-4a5a-a4e7-4c54b009fad6)